automation/src/test/java/org/greenplum/pxf/automation/features/avro/HdfsWritableAvroTest.java

```diff
@@ -24,8 +24,21 @@
 
     private ReadableExternalTable readableExternalTable;
     private ArrayList<File> filesToDelete;
-    private final String[] avroPrimitiveTableCols = new String[]{
+    private final String[] avroPrimitiveWritableTableCols = new String[]{
```

Comment:
these should be static final String arrays
```suggestion
    private static final String[] AVRO_PRIMITIVE_WRITABLE_TABLE_COLS = new String[]{
```